plover/assets/main.json

```diff
@@ -25411,7 +25414,10 @@
 "HOPBLG/KEUPB/AOES": "Hodgkin's",
 "HOPBLG/POPBLG": "hodgepodge",
 "HOPBLG/SKWRE/TPHAEUT": "homogeneity",
+"HOPBLS": "honestly",
+"HOPBS": "honest",
```

Comment:
These are too "briefy" for me. Can you change them to `H*OPBS` so that we are following the convention where `*S == st`?